reference/storefront_tokens.v3.yml

```diff
@@ -169,18 +174,22 @@ components:
       x-internal: false
       properties:
         channel_id:
-          type: integer
-          minimum: 1
-          description: Channel ID for requested token
-          example: 1
+          oneOf:
+           - type: integer
+             example: 1
+           - type: array
+             items:
+               type: integer
+             example: [667251, 1]
+          description: Channel ID or list of channel IDs for requested token
         expires_at:
           type: integer
           description: Unix timestamp (UTC time) defining when the token should expire. Supports seconds, but does not support milliseconds, microseconds, or nanoseconds.
           example: 1885635176
           minimum: 0
       required:
-        - channel_id
         - expires_at
+        - channel_id
```

Comment:
ok I was not clear enough 
we require either `channel_ids` or `channel_id` . `channel_ids` is of array type, `channel_id` is integer type. 

Let mention only channel_ids then